scss/mixins/_utilities.scss

```diff
@@ -1,6 +1,6 @@
 // Utility generator
 // Used to generate utilities & print utilities
-@mixin generate-utility($utility, $infix) {
+@mixin generate-utility($utility, $infix, $isRfsMediaQuery: false) {
```

Comment:
Is there an intention for the variable name of uppercase? I'm concerned about the consistency.

Reply:
Good call, changed it to kebab case